### ISSUE TITLE
Feature: Accents for Held keys?

### ISSUE THREAD

**Issue by rbreaves** [mooz/xkeysnail#127]
I was thinking this would be better suited for another application - but then again xkeysnail does intercept every key press and technically could detect when a key is held (repeating). I am wondering what others here think about implementing a method to show an overlay popup when a letter is held for accent options?

https://www.reddit.com/r/gnome/comments/bwofdu/is_there_a_way_to_quickly_type_accents_in_gnome/

**Comment by luizoti** [comment]
Gtk or Qt probably can create the popup, probably is possible.

**Comment by joshgoebel** [comment]
> I was thinking this would be better suited for another application 

Agreed.  Any type of UI really feels out of scope to me personally.
